sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamTest.scala

```diff
@@ -871,6 +871,9 @@ trait StreamTest extends QueryTest with SharedSparkSession with TimeLimits with
 
           case r if r < 0.7 => // AddData
             addRandomData()
+            // Must check data after adding data in case we delete the topic with added data
```

Comment:
It seems odd to explain Kafka specific one here, as the logic is a generic one. Even we need to infer which module/suite needs this with the only one hint "topic".

If we'd like to explain Kafka specific issue here, let's make clear which suite requires it.

Reply:
Updated. Thanks.